deploy/kubernetes/setup-csi-snapshotter.yaml

```diff
@@ -89,6 +89,7 @@ spec:
           args:
             - "--csi-address=$(ADDRESS)"
             - "--connection-timeout=15s"
+            - "--leader-election=true"
```

Comment:
By default leader-election should be false in the deployment yaml file.  User can change to true when needed.

Reply:
ah okay, I was trying to be consistent with examples in other sidecars, they have leader election enabled in all the examples